eclair-node/src/main/scala/fr/acinq/eclair/api/Service.scala

```diff
@@ -230,6 +230,12 @@ trait Service extends ExtraDirectives with Logging {
                             complete(eclairApi.findRoute(nodeId, amount))
                           }
                         } ~
+                        path("findroutebetweennodes") {
+                          formFields("sourceNodeId".as[PublicKey], "targetNodeId".as[PublicKey], amountMsatFormParam) {
+                            (sourceNodeId, targetNodeId, amount) =>
+                            complete(eclairApi.findRouteBetween(sourceNodeId, targetNodeId, amount))
```

Comment:
nit: indentation is a bit weird here?
```suggestion
                            (sourceNodeId, targetNodeId, amount) => complete(eclairApi.findRouteBetween(sourceNodeId, targetNodeId, amount))
```